media-gfx/xpaint/files/xpaint-2.10.2-respect-ldflags.patch

```diff
@@ -0,0 +1,15 @@
+--- a/util/Makefile	2017-03-14 20:54:48.000000000 +0200
++++ b/util/Makefile	2021-03-14 13:48:15.150984967 +0200
+@@ -10,10 +10,10 @@
+ endif
+ 
+ pdfconcat:
+-	$(CC) $(CFLAGS) -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
++	$(CC) $(CFLAGS) $(LDFLAGS) -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
```

Comment:
```suggestion
+	$(CC) $(CFLAGS) $(LDFLAGS) -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
```

We don't want to force -O3 or to strip.

Reply:
`-O3` and `-s` were already stripped with *sed* from the ebuild.  But yes, I'll move them here

Reply:
Yeah, it's just nicer to do a patch where we can (especially given we're already doing it) as sed can 'fail silently'.